asteroidsyncserviced/bluez/bluezclient.cpp

```diff
@@ -19,28 +19,50 @@
 
 #include "bluezclient.h"
 #include "dbus-shared.h"
-#include "../libasteroid/services/common.h"
+#include "services/common.h"
+
+#include "bluez_helper.h"
+#include "freedesktop_objectmanager.h"
+#include "freedesktop_properties.h"
+#include "bluez_adapter1.h"
+#include "bluez_agentmanager1.h"
 
 #include <QDBusConnection>
 #include <QDBusReply>
 #include <QDebug>
+#include <stdexcept>
+
+class BluezClientImpl {
```

Comment:
The design-pattern you're using here is the "opaque pointer" https://en.wikipedia.org/wiki/Opaque_pointer#C++
The "Impl" suffix does not seem appropriate because this does not actually implement anything. This is just a glorified storage space.
If you like that pattern, what about following the Qt Project's best practices https://wiki.qt.io/D-Pointer ? Typically, in the Qt codebase, this would be called BluezClientPrivate.

Reply:
Thanks, I did not know about that Qt best practice and have now adopted it.